hw3/tests/test_hw3_t02.py

```diff
@@ -0,0 +1,508 @@
+    2035,
+    2044,
+    2044,
+    2046,
+    2047,
+    2048,
+    2051,
+    2053,
+    2055,
+    2062,
+    2063,
+    2063,
+    2064,
+    2066,
+    2066,
+    2067,
+    2067,
+    2070,
+    2070,
+    2070,
+    2072,
+    2076,
+    2078,
+    2078,
+    2080,
+    2081,
+    2082,
+    2082,
+    2082,
+    2082,
+    2083,
+    2083,
+    2087,
+    2089,
+    2089,
+    2091,
+    2092,
+    2093,
+    2093,
+    2094,
+    2097,
+    2100,
+    2100,
+    2100,
+    2101,
+    2103,
+    2104,
+    2108,
+    2110,
+    2112,
+    2114,
+    2115,
+    2123,
+    2125,
+    2125,
+    2126,
+    2127,
+    2130,
+    2131,
+    2133,
+    2135,
+    2138,
+    2140,
+    2147,
+    2148,
+    2149,
+    2149,
+    2151,
+    2152,
+    2153,
+    2153,
+    2153,
+    2154,
+    2157,
+    2159,
+    2159,
+    2160,
+    2160,
+    2160,
+    2163,
+    2164,
+    2165,
+    2165,
+    2166,
+    2166,
+    2167,
+    2171,
+    2172,
+    2173,
+    2174,
+    2174,
+    2174,
+    2176,
+    2178,
+    2181,
+    2182,
+    2183,
+    2184,
+    2184,
+    2184,
+    2187,
+    2188,
+    2191,
+    2191,
+    2192,
+    2200,
+    2200,
+    2202,
+    2202,
+    2204,
+    2206,
+    2207,
+    2211,
+    2212,
+    2214,
+    2215,
+    2215,
+    2217,
+    2217,
+    2219,
+    2219,
+    2221,
+    2223,
+    2223,
+    2225,
+    2225,
+    2231,
+    2232,
+    2232,
+    2233,
+    2234,
+    2238,
+    2239,
+    2239,
+    2242,
+    2245,
+    2246,
+    2247,
+    2247,
+    2249,
+    2251,
+    2261,
+    2269,
+    2269,
+    2271,
+    2274,
+    2281,
+    2281,
+    2281,
+    2282,
+    2282,
+    2282,
+    2285,
+    2286,
+    2287,
+    2298,
+    2298,
+    2303,
+    2305,
+    2305,
+    2310,
+    2310,
+    2313,
+    2320,
+    2322,
+    2325,
+    2325,
+    2325,
+    2326,
+    2326,
+    2327,
+    2327,
+    2329,
+    2329,
+    2332,
+    2333,
+    2337,
+    2339,
+    2342,
+    2343,
+    2349,
+    2349,
+    2350,
+    2350,
+    2357,
+    2358,
+    2362,
+    2368,
+    2373,
+    2375,
+    2377,
+    2380,
+    2381,
+    2383,
+    2383,
+    2385,
+    2396,
+    2396,
+    2400,
+    2404,
+    2408,
+    2420,
+    2421,
+    2429,
+    2436,
+    2438,
+    2439,
+    2441,
+    2441,
+    2447,
+    2448,
+    2450,
+    2457,
+    2459,
+    2466,
+    2468,
+    2470,
+    2473,
+    2475,
+    2481,
+    2482,
+    2483,
+    2487,
+    2497,
+    2500,
+    2501,
+    2504,
+    2508,
+    2517,
+    2528,
+    2529,
+    2541,
+    2543,
+    2543,
+    2545,
+    2549,
+    2557,
+    2565,
+    2574,
+    2580,
+    2589,
+    2609,
+    2611,
+    2619,
+    2623,
+    2650,
+    2650,
+    2653,
+    2659,
+    2664,
+    2665,
+    2680,
+    2681,
+    2698,
+    2700,
+    2709,
+    2740,
+    2750,
+    2763,
+    2766,
+    2785,
+    2810,
+]
+
+
+def test_slow_calc_500_numbers():
+    assert sorted(slow_calc_500_numbers()) == pre_calculated_data_sorted
```

Comment:
What was the task requirement?

Reply:
To calculate the total sum

Reply:
Sometimes my question is the only question, sometimes it is a call to action. 

Does your function calculate the sum?

Reply:
Yeah, I corrected my code. I'll push that changes after figuring out a number of processes.